res/database/Default/user/StukovAds.xml

```diff
@@ -0,0 +1,807 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tvtdb>
+	<version value="3" />
+	<!--
+	0			0 	 - 0,25			100 - 3000
+	0,25		0,10 - 0,60			100 - 2500
+	0,5			0,33 - 1,20			100 - 2000
+	1			0,75 - 2,75			100 - 1750
+	2,5			2,25 - 5,25			100 - 1500
+	5			4,75 - 8,00			100 - 1250
+	10			6,00 - 12,00		100 - 1200
+	25			9,00 - 18,00		100 - 1000
+	50			12,00 - 24,00		100 - 850
+	100			20,00 - 50,00		100 - 750
+	-->
+	<allads>
+
+		<ad id="Stukov-Logical1">
+			<title>
+				<de>Logikus</de> <!-- orig: Logical -->
+			</title>
+			<description>
+				<de>Der Super-Arcade-Spaß von RegenbogenWünsche!</de>
+				<en>Super arcade fun by RainbowWishes!</en>
+			</description>
+			<conditions min_audience="0.3951" min_image="0" target_group="0" pro_pressure_groups="0" contra_pressure_groups="0"/>
+			<data quality="25" repetitions="5" duration="8" profit="1438" penalty="3131" fix_infomercial_profit="1"/>
```

Comment:
The audience+repetitions+duration values should be checked in the game. In some cases the duration seems quite long. Really 8 days?
Also, I am not sure if too many ads should be added. In particular for day time and "trash" ads it is part of the "fun" of the game the fitting ads are scarce. If all audience ranges are available at all times, there is little pressure for ensuring ahead of time that one is not running out of suitable contracts.

Also it is questionable, if different contract types for the same "Company" (at least for som many) should be available simultaneously. These additions need a more thorough check.

Reply:
Yeah the Stukov ads seem a bit "too good".

> Also, I am not sure if too many ads should be added

There were some complaints in the forums about not getting suitable ads. The reason might be, for now, that they aren't sufficient yet in number. There might be some "hot spots" because the progressed players need them for their night program and the less progressed ones for their mid-day program as well.

In general, long-term I don't think there should be restrictions on the number of ads, we should be able to add as many as we like. How and which ads are then made available would be a question of balancing etc, the game could "artificially" restrict certain slots if desired.

Reply:
I see your point that the restricting the number of (suitable) ads should not be done in the database but in the game itself. So it remains to check the individual ads, whether they are too good compared to the existing ones (profit,repititions,duration).

Maybe instead of changing the titles and texts, the availability should be made disjoint, as you can restrict the years when an ad is available. Company starts small (cheap ad), grows and prospers (better ads), and goes into decline, again..,

As this file is completely independent of the rest (unlike roles/persons and programmes/scripts), it could be separated and discussed in a different PR.

Reply:
I have gone over this file and removed all the dupes and replaced them with hitherto unused Mad TV ones and new ones.

I'll keep it in the PR, playtesting feels 'okay', fine-tuning can be done at a later point.

> Really 8 days?

While some others are are quite demanding, only 1 day, requiring many broadcasts. (And Mad TV had 1-2 that went for 7 days or so too.) So I guess it was designed on full purpose that way.

Reply:
But in Mad TV there were only prime time slots (which makes many things much easier). I have reviewed the existing ads (#1057) and added a statistic export that will make evaluating these changes easier. You'll get more specific feedback on this file...